source/PluginHelper.cpp

```diff
@@ -0,0 +1,198 @@
+/* PluginHelper.cpp
+Copyright (c) 2023 by RisingLeaf(https://github.com/RisingLeaf)
+
+Endless Sky is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later version.
+
+Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+this program. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#include "PluginHelper.h"
+
+#if defined _WIN32
+#define WIN32_LEAN_AND_MEAN
```

Comment:
Move this below where you include Windows.h

Reply:
did that

Reply:
moving this breaks the checks